From 381159323dc226f7afc5b8dc019ec20b87ef5564 Mon Sep 17 00:00:00 2001
From: Avery Pennarun <apenwarr@gmail.com>
Date: Thu, 15 May 2014 01:27:20 -0400
Subject: [PATCH] mac80211: disable setting QoS flags from DSCP values.

This causes some problems with the current ath10k driver, which seems to do
wrong things with ADDBA if multiple QoS TIDs are in use at once.  In any
case, it's not clear that the default settings were what we wanted, so this
shouldn't hurt performance for now.  Eventually we can remove this when the
ath10k driver has been fixed up.
---
 net/wireless/util.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/net/wireless/util.c b/net/wireless/util.c
index e5872ff..0193e24 100644
--- a/net/wireless/util.c
+++ b/net/wireless/util.c
@@ -695,6 +695,7 @@ unsigned int cfg80211_classify8021d(struct sk_buff *skb,
 {
 	unsigned int dscp;
 	unsigned char vlan_priority;
+	return 0;
 
 	/* skb->priority values from 256->263 are magic values to
 	 * directly indicate a specific 802.1d priority.  This is used
-- 
1.9.1.423.g4596e3a

